### PR TITLE
Dark theme markdown fixes

### PR DESCRIPTION
**Pull request by Cherrg**
* fixes tables in dark theme
* fix simpleMDE preview in dark theme

## Images

|                                                | OLD          | NEW        |
|:------------------------------------- |:-------------:|:-------------:|
| Tables in Markdown <br> Commit 1              | grafik  | grafik  |
| Markdown editor wiki + repo  <br> Commit 2  | grafik  | grafik |

Signed-off-by: Michael Gnehr <michael@gnehr.de>

### CODE REVIEW

**Comment by silverwind** [comment]
The scrollbar styling is Firefox-only as of now. Maybe we want to add site-wide dark scrollbars to the dark theme?

See [here](https://github.com/StylishThemes/Overlay-Scrollbars/blob/master/global-overlay-scrollbars.user.css) for cross-browser scrollbar styling.

**Comment by silverwind** [comment]
The scrollbar styling is Firefox-only as of now. Maybe we want to add site-wide dark scrollbars to the dark theme?

See [here](https://github.com/StylishThemes/Overlay-Scrollbars/blob/master/global-overlay-scrollbars.user.css) for cross-browser scrollbar styling.

**Comment by lafriks** [comment]
@silverwind that should probably be done in other PR

**Comment by lafriks** [comment]
@silverwind that should probably be done in other PR

**Comment by silverwind** [comment]
Then I suggest removing this non-portable scrollbar style here.

File: public/less/themes/arc-green.less [hunk 1]
```diff
@@ -501,7 +501,36 @@ a.ui.basic.green.label:hover {
...
+
+.repository.file.editor.edit,
+.repository.wiki.new .CodeMirror {
+    border-right: 1px solid rgba(187,187,187, 0.6);
+    border-left: 1px solid rgba(187,187,187, 0.6);
+    border-bottom: 1px solid rgba(187,187,187, 0.6);
+
+    .editor-preview {
+        background: #353945;
+        scrollbar-color: #2a2a2a white;
```
Comment (lafriks) [inline public/less/themes/arc-green.less]:
```suggestion
```

File: public/css/theme-arc-green.css [hunk 2]
```diff
@@ -92,7 +92,13 @@ footer{background:#2e323e;border-top:1px solid #313131}
 .markdown:not(code) h2{border-bottom:1px solid #304251}
 .hljs,.hljs-keyword,.hljs-selector-tag,.hljs-subst{color:#9daccc}
 .markdown:not(code) .highlight pre,.markdown:not(code) pre{background-color:#2a2e3a;border:1px solid #404552}
-.markdown:not(code) table tr:nth-child(2n){background-color:#474d61}
+.markdown:not(code) table tr:nth-child(2n){background-color:#2a2e39}
+.markdown:not(code) table tr:nth-child(2n-1){background-color:#383b44}
+.markdown:not(code) table thead tr:nth-child(2n-1){background-color:#464c5d!important}
+.markdown:not(code) table td,.markdown:not(code) table th{border-color:#4c505c!important}
+.repository.file.editor.edit,.repository.wiki.new .CodeMirror{border-right:1px solid rgba(187,187,187,.6);border-left:1px solid rgba(187,187,187,.6);border-bottom:1px solid rgba(187,187,187,.6)}
+.repository.file.editor.edit .editor-preview,.repository.wiki.new .CodeMirror .editor-preview{background:#353945;scrollbar-color:#2a2a2a #fff}
```
Comment (lafriks) [inline public/css/theme-arc-green.css]:
```suggestion
.repository.file.editor.edit .editor-preview,.repository.wiki.new .CodeMirror .editor-preview{background:#353945}
```

**Review by lafriks — CHANGES_REQUESTED** [review]
This should be globally added to all scrollbars and made portable but I would better see this done in other PR

**Comment by Cherrg** [comment]
I will remove the FF scroolbar line there. 
I had added FF Scrollbar, as Chrome already got an colored scrollbar by semantic.css. FF was missing. (And Scrollbar was last thing with wrong color.)
You right lets do this globally on different PR.

File: public/css/theme-arc-green.css (hunk 2 above)
Comment (Cherrg) [reply public/css/theme-arc-green.css]: Done in other Commit

**Comment by Cherrg** [comment]
done, even created seperate PR for dark theme scrollbars: #7269

**Review by lafriks — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Comment by Cherrg** [comment]
Please restart drone build. Ty

**Pull request merged**